connectivity/mbedtls/include/mbedtls/aes.h

```diff
@@ -20,7 +20,8 @@
  *          <https://ieeexplore.ieee.org/servlet/opac?punumber=4375278>.
  */
 
-/*  Copyright (C) 2006-2018, Arm Limited (or its affiliates), All Rights Reserved.
+/*
+ *  Copyright The Mbed TLS Contributors
```

Comment:
@0xc0170 This copyright change was recently made in all files in Mbed TLS. Since the code is imported, are we okay with this?